src/main/webapp/pages/common/banner.jsp

```diff
@@ -92,18 +94,33 @@ function getCsrfToken() {
  		<div style="display: block">
 			<%
 				UserSessionData usd = null;
+				NotificationService ns = new NotificationService(); 
 				if (request.getSession().getAttribute(IActionConstants.USER_SESSION_DATA) != null) {
 					usd = (UserSessionData) request.getSession().getAttribute(IActionConstants.USER_SESSION_DATA);
+					String saveDate = ns.getLoginFromCombinedId(usd.getLoginName()).toString();
```

Comment:
better to just pass this object via a session attribute or you you can add it to the existing  USER_SESSION_DATA attribute